README.md

```diff
@@ -1,6 +1,106 @@
+# Logstash Filter Aggregate Documentation
```

Comment:
I just created a separate file named `BUILD.md`. It is pushed on github.
If you think about a better file name, tell me.